frontend/src/scenes/web-analytics/tiles/WebAnalyticsTile.tsx

```diff
@@ -367,6 +376,18 @@ export const webAnalyticsDataTableQueryContext: QueryContext = {
         action_name: {
             title: 'Action',
         },
+        replay_url: {
```

Comment:
do we want to conditionally add the column so we can A/B test it? 
or just add it and track user behavior with insights/replay?

(open question, no push in either direction from me)